test_utils/src/relay.rs

```diff
@@ -96,18 +96,25 @@ impl<'a> Relay<'a> {
         &self,
         client_id: u64,
         subscription_id: &nostr::SubscriptionId,
-        filters: &[nostr::Filter],
+        // TODO: enable filters
+        _filters: &[nostr::Filter],
     ) -> Result<bool> {
         // let t: Vec<nostr::Kind> = self.events.iter().map(|e| e.kind).collect();
         // .filter(|e| filters.iter().any(|filter| filter.match_event(e)))
         // println!("letsgo{:?}", t);
+
         self.respond_events(
             client_id,
             subscription_id,
             &self
                 .events
                 .iter()
-                .filter(|e| filters.iter().any(|filter| filter.match_event(e)))
+                // FIXME:
+                // `filter.match_events` does not exist anymore
+                // it has been moved to `nostr_database_::FilterIndex`
+                // but it's private now
+                // .filter(|e| filters.iter().any(|filter|filter.match_event(e)))
+                .filter(|_| true)
```

Comment:
`match_events` and other functions have been moved from `Filter` to `nostr_database_::FilterIndex` - see https://github.com/rust-nostr/nostr/commit/2439a3dee83da819ab824a7b7f37e6b23f6b3c5d

Unfortunately `match_events` is private now.